apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -375,20 +375,30 @@ const OutOfOfficePage = () => {
   }, [openModalOnStart]);
 
   const [openModal, setOpenModal] = useState(false);
+
+  const { data } = trpc.viewer.outOfOfficeReasonList.useQuery();
+  // TODO: The best solution is we should find the cause of full page flicker
+  // Taling about hydration problem, migrating to appDir might solve the problem automatically
+  // which will reset the Modal state, which makes bad UX and fails (superhuman click speed) E2E tests
```

Comment:
Don't add too many comments; only include them if there's a TODO

Reply:
I'm on it :smile: